README.md

```diff
@@ -2,47 +2,52 @@
+  * In order to create through the interactive menu issue the following command:  
+	```node . create --verbose```
+  * In order to deploy polkadot using a the preset configuration file: my-local-testnet.json and issue the following command:  
+	```node . create --config config/create.local.sample.json --verbose```
+	The process will start creating an instance of polkadot inside a your local kubernetes cluster that will be created as part of the procedure using [kubernetes-sigs/kind](https://github.com/kubernetes-sigs/kind). The entire procedure will take some time, so it might be a good idea to get some coffee at this point.
+
+
+3. Once the local cluster is created, a kubeconfig file will be created in ~/.kube/ with a prefix of kind-config followed by the name of the created cluster (eg: kind-config-my-demo-testnet). This file should be moved or copied to your local .kube directory. 
+
+4. Attach kubectl to this kubeconfig file by issuing the command: ```export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"```  You can verify your installation by using the following commands:  ```kubectl cluster-info```  to see an overview of your local installation and ```kubectl get pods``` to see the pods running on your local cluster. You can also view all your local deployments using the command: ```node . list```
+
+5. At this point you can attach to the local polkadot web socket by visiting the websockets endpoint available at ws://127.0.0.1:11000 Furthermore you at this point you will be presented with the raw seeds for the created accounts, including the nodeKey, peerId, stash address and seed etc.
+
+6. Once you are done with your local deployment of polkadot, you can delete your deployment using the destroy [name] command: ```node . destroy testnet5```
+More information on the polkadot-deployer usage commands can be found in the [usage](#usage) section.
 
-Open a terminal and run this command:
 
-```
-$ npm i -g polkadot-deployer
-```
-Check that all is correct:
-```
-$ polkadot-deployer --version
-```
-You can get an overview of the available actions with:
-```
-$ polkadot-deployer --help
-```
 Check the [Troubleshooting section](#troubleshooting) if something goes wrong with the installation.
 
 ## Remote deployments
 
-polkadot-deployer allows you to create remote cloud deployments, currently it
-supports Google Cloud Platform for the infrastructure and Cloudflare for the
-DNS settings that make your network accessible through websockets RPC.
 
-In order to be able to deploy remotely you will need:
+To perform a remote deployment of polkador to a public cloud provider we will follow the same general path. We will need to specify a number of extra attributes inside the json file with  extra information such as our credentials etc. In order to be able to deploy remotely you will need:
 
-* A project on GCP.
+* A Linux machine to run this tool (macOS may fail).
```

Comment:
I would add a link here to the troubleshooting section below.
```suggestion
* A Linux machine to run this tool (macOS may fail, see the [Troubleshooting section](#troubleshooting) in case you have problems running the tool).
```